proto/resource_manager.proto

```diff
@@ -61,6 +61,8 @@ message TokenBucketsRequest {
   repeated TokenBucketRequest requests = 1;
   uint64 target_request_period_ms = 2;
   uint64 client_unique_id = 3;
+  // label background request.
+  bool is_background = 4;
```

Comment:
Is it better to put this field into TokenBucketRequest? We may want to batch both background/foreground request together

Reply:
cc @HuSharp  :)

Reply:
Okay, I opened a new pr to change it.https://github.com/pingcap/kvproto/pull/1161